oauth2_provider/tests/test_models.py

```diff
@@ -41,6 +41,77 @@ def test_allow_scopes(self):
         self.assertTrue(access_token.allow_scopes(['write', 'read', 'read']))
         self.assertTrue(access_token.allow_scopes([]))
         self.assertFalse(access_token.allow_scopes(['write', 'destroy']))
+        self.assertTrue(access_token.allow_scopes('read write'))
```

Comment:
This test is a great candidate for [parameterization](http://doc.pytest.org/en/latest/parametrize.html).